authors/apps/articles/templates/comments.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
for this html you can have a base template and inherit from it rather than repeat code